common/src/testFixtures/kotlin/org/onflow/flow/common/test/AbstractFlowEmulatorExtension.kt

```diff
@@ -55,7 +55,7 @@ annotation class FlowServiceAccountCredentials
 @Inherited
 @API(status = API.Status.STABLE, since = "5.0")
 annotation class FlowTestAccount(
-    val signAlgo: SignatureAlgorithm = SignatureAlgorithm.ECDSA_SECP256k1,
```

Comment:
Just making sure this is not changed because there is an issue with using the first curve?

Reply:
Hi @tarakby I have reverted this change, tests are passing with both curves as we no longer hardcode the signAlgo in the Cadence scripts. Thanks!